src/ui/Commands.java

```diff
@@ -46,7 +46,8 @@ of this software and associated documentation files (the "Software"), to deal
 	
 	// Video page commands
 	static final Command watchCmd = new Command(Locale.s(CMD_Watch), Command.OK, 10);
-	static final Command downloadCmd = new Command(Locale.s(CMD_Download), Command.SCREEN, 9);
+	static final Command downloadCmd = new Command(Locale.s(CMD_Download), Command.SCREEN, 8);
+	static final Command showLinkCmd = new Command(Locale.s(CMD_ShowLink), Command.SCREEN, 6);
```

Comment:
Is link showing more important than downloading? Now it has higher priority.

Reply:
Idk how it works, but it really seems randomly-sorted on nokia